### DISCUSSION TITLE
Creating repo

### DISCUSSION THREAD

**Discussion by unknownguy1805** [community/community#132990] [New to GitHub] [answered]
How do I create a new repository on GitHub?

**Comment by unknownguy49** [comment, marked as answer]
Click the "+" icon in the upper-right corner and select "New repository". Name your repository and add a description if you want. Choose to make it public or private. Also you can add README file, LICENSE file and a .gitignore file. Then click "Create repository" and there you go.

**Comment by Ammar-Alnagar** [comment]
You can also use git init and then publish the repo
